### ISSUE TITLE
The cursor will go to the screen bottom when I touch the panel top position. Macbook 2015 13.3

### ISSUE THREAD

**Issue by kazenoai** [imbushuo/mac-precision-touchpad#134]
The cursor will to to the screen's bottom immediately when I figure touch the top position of touch panel.
Computer: Macbook 2015 13.3
OS: Windows 10, 1803
Driver type: SPI driver(07/14/2018,14.56.52.89)

**Comment by kazenoai** [comment]
<img width="1275" alt="default" src="https://user-images.githubusercontent.com/43717077/46323269-1649f980-c620-11e8-91b6-fdfae3d22460.PNG">
I change the code of input.c, and try it in my computer. The cursor does not lost and appear at screen's bottom area when I touch the top area of touch panel. And this issue also will occur when I touch the touch panel left side, and the cursor will appear the right side edge of screen.

**Comment by RLst** [comment]
Yes I think this happens to me too. I have a MacBook Pro retina late 2013. When using it would seem to randomly teleport to the bottom or around bottom right corner of the screen and then ends up opening some other program. Extremely annoying

**Comment by kazenoai** [comment]
@RLst You can find the position of that code(SPI: input.c, USB: InputInterrupt.c), and try to modify the code and run the driver on your computer.

**Comment by imbushuo** [comment]
@kazenoai Can you send a PR to me?

**Comment by kazenoai** [comment]
I have sent  PR to you! Your project is great. Thank you.
